README.md

```diff
@@ -71,14 +71,14 @@ be the latest.
 This list presents the known working version combos between the target operating system
 and Erlang/OTP.
 
-| Operating system | Erlang/OTP  | Status
-|-                 |-            |-
-| `ubuntu-18.04`   | 17.0 - 25.3 | ✅
-| `ubuntu-20.04`   | 21.0 - 27   | ✅
-| `ubuntu-22.04`   | 24.2 - 27   | ✅
-| `ubuntu-24.04`   | 24.3 - 27   | ✅
-| `windows-2019`   | 21* - 25    | ✅
-| `windows-2022`   | 21* - 27    | ✅
+| Operating system | Erlang/OTP   | OTP Architecture | Status
+|-                 |-             |                  |-
+| `ubuntu-18.04`   | 17.0 - 25.3  | 32 bit           | ✅
```

Comment:
I believe we are getting Ubuntu from https://github.com/hexpm/bob?tab=readme-ov-file#erlang-builds and we don’t have 32bit builds.

Instead of 32bit/64bit perhaps it is more useful to mention x86, x86_64 and arm64 given we have former two for Windows and latter two for Ubuntu (and eventually macOS)

Reply:
Sorry about the incorrect 32-bit arch for Ubuntu. I have done what you suggested.